src/commands/check/index.ts

```diff
@@ -17,57 +26,117 @@ interface Ctx {
       cargoDylint?: string | null;
       cargoContract?: string | null;
     };
+    supportedInk?: string;
+    missingTools: string[];
     contracts: Record<string, Record<string, string>>;
-    node?: string | null;
+    swankyNode: string | null;
   };
   swankyConfig?: SwankyConfig;
-  mismatchedVersions?: Record<string, string>;
+  mismatchedVersions: Record<string, string>;
   looseDefinitionDetected: boolean;
 }
 
 export default class Check extends SwankyCommand<typeof Check> {
   static description = "Check installed package versions and compatibility";
 
+  static flags = {
+    print: Flags.string({
+      char: "p",
```

Comment:
why `p`? 'path'? I guess 'o' is more common

Reply:
you're right, bienvu!